tests/e2e/autoscaling/00-create-vm.yaml

```diff
@@ -0,0 +1,146 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- ../../../deploy/autoscale-scheduler.yaml
+- ../../../deploy/autoscaler-agent.yaml
```

Comment:
At first glance, I feel like this should be handled in `make deploy`, not here. Thoughts? Maybe there's something about kuttl that means this makes more sense?